src/components/UI/Display/MAPI/AccessControl/AccessControlSubjectSetItem.tsx

```diff
@@ -34,11 +34,16 @@ const StyledAnchor = styled(Anchor)`
   }
 `;
 
+const StyledDrop = styled(Drop)`
+  z-index: 1071; /* To appear above the Bootstrap Tooltip, which has a z-index of 1070 */
```

Comment:
That's very specific 😆

Reply:
I know 😅 it's probably not the smartest solution, so definitely let me know if there's a better way!